### DISCUSSION TITLE
Sortable and animated together

### DISCUSSION THREAD

**Discussion by stla** [JedWatson/react-select#4911] [Q&A]
Hello,

  I'm not able to get "animated" working when I use sortable labels. I tried to replace `MultiValue` with `SortableMultiValue` and           `MultiValueLabel` with `SortableMultiValueLabel` in the animated components but that does not work.

**Comment by ebonow** [comment]
Would you mind sharing a codesandbox?

**Comment by stla** [comment]
Yes, I will create an account. Please be patient.

**Reply by ebonow** [reply]
Here's a template to get you started.

https://codesandbox.io/s/react-select-sortable-handle-hoc-ipud9?file=/example.js

This was a working draft I used while I worked on fixing an issue with the sortableHOC example in the documentation so it should hopefully be a good start.
